documents/src/pages/utils/tree-manager.md

```diff
@@ -0,0 +1,50 @@
+<!-- 
+title: Tree Manager
+location: ./custom-components/utils/tree-manager
+type: page
+layout: default
+language_tabs: [javascript, typescript]
+-->
+
+# Tree Manager
+
+Tree Manager is a simplified version of [Collection Composer](/custom-components/utils/data-management#collection-composer) providing an ability to manage nested items in [Tree](/elements/tree)/[Tree Select](/elements/tree-select) component with APIs for reading and updating item properties.
+
+@> Tree Manager supports type generic defining the type of its items. In this document, this generic type is noted as `T`.
+
+```javascript
+// Select the first item in Tree.
+const tree = document.querySelector('ef-tree');
+const data = [ 
+  { label: 'one', value: '1'}, 
+  { label: 'two', value: '2', icon: 'info' }
+];
```

Comment:
The trailing spaces of each line of code make a bad DX when a developer copy and ESLint show warning message